common/src/models/Transaction.ts

```diff
@@ -6,7 +6,11 @@ export enum TransactionType {
   BUY = 'buy',
   SELL = 'sell',
   DIVIDENDS = 'dividends',
+  DISTRIBUTION = 'distribution',
+  DRIP = 'drip',
```

Comment:
What are these transactions used for?

Reply:
A distribution is similar to a dividend, but have different fiscal impact
---
- A DRIP is a combinaison between a Dividend and a BUY.

Ex: Stock is at 10$ and you have a 25$ dividend, your broker can on your behalf instead of giving you 25$ and you then buy 2 stocks ( and pay fees), your broker will buy the two stocks, and deposit 5$ in your account.

- First you save the Brooker fees
- And the stock is bought immediately, so you will generate more compound interest.
- Also, its automatic, so in this exame, every month you get 2 new.shares

https://www.investopedia.com/ask/answers/what-is-a-drip/